src/ucm/cuda/cudamem.c

```diff
@@ -206,13 +210,38 @@ UCM_CUDA_FREE_FUNC(cuMemFreeAsync, UCS_MEMORY_TYPE_CUDA, CUresult, arg0, 0,
                    "ptr=0x%llx, stream=%p", CUdeviceptr, CUstream)
 #endif
 
+CUresult ucm_cuModuleGetGlobal_v2(CUdeviceptr *dptr, size_t *bytes,
+                                  CUmodule hmod, const char *name)
+{
+    CUresult ret;
+    size_t size;
+    size_t *size_ptr;
+
+    if (dptr == NULL) {
+        return ucm_orig_cuModuleGetGlobal_v2(dptr, bytes, hmod, name);
```

Comment:
add comment:
/* Device pointer is not returned */

Reply:
Added.